app/app.js

```diff
@@ -147,13 +160,12 @@ angular.module('app', ['angular-md5'])
        * above that we want to expand a different one, this block maintains
        * the expanded box's position on the page by scrolling the viewport.
        */
-      
       if (isForceScroll) { //Need Klaus to explain how this works before I tackle this
         $('html, body').animate({
-          scrollLeft: currentComic.containerStyles.left - 200,
-          //scrollLeft: currentComic.containerStyles.left - leftOffset,
-          scrollTop:  currentComic.containerStyles.top + 300
-          //scrollTop:  currentComic.containerStyles.top + topOffset
+          //scrollLeft: currentComic.containerStyles.left - 200,
```

Comment:
It's easier for me to read diffs if code isn't commented out - it confuses the diff engine if this is done

Reply:
I've removed the code in a later commit